index.html

```diff
@@ -4285,6 +4285,23 @@ <h1>DID Methods</h1>
             <a href="https://github.com/PML-ID/pml-did-specs/blob/main/did-method.md">PML DID Method</a>
           </td>
         </tr>
+        <tr>
+          <td>
+            did:sideos:
```

Comment:
Should be in alphabetical order in the list.  (The other two trailing ones are in the wrong spot and should also be moved.)

Reply:
done. sorted the complete list.

Reply:
Nice.  Guess there were some other ordering issues too.